src/daemon.rs

```diff
@@ -39,7 +38,7 @@ pub fn watchdog_enabled(unset_env: bool) -> Option<time::Duration> {
 
     let pid = {
         if let Some(pid_str) = env_pid {
-            if let Ok(p) = pid_str.parse::<pid_t>() {
+            if let Ok(p) = pid_str.parse() {
```

Comment:
Unrelated, but after the actual change from this MR this was the last instead of direct `libc` access, so I figured I'd just quickly remove it.

Reply:
On second thought, this change really doesn't belong here, so I removed it agaiin.